README.md

```diff
@@ -145,7 +153,9 @@ Chrome is pre-installed so that Django's selenium tests can be run in headless
 mode with a virtual display (id `:99`). For example, you may run a specific test
 like so:
 
-    (vm) $ runtests27-sqlite3 admin_widgets --selenium chrome
+    (vm) $ runtests36-sqlite3 admin_widgets --selenium chrome --parallel 1
+
+The test suite will sometimes hang when running selenium tests in parallel mode.
```

Comment:
Reproduced without the box. I'm suspecting https://github.com/django/django/commit/bece837829eafbc22f2598dadf82c9a8364b085a might be the culprit but am running repeated tests on the commit before to see if reproduces there.